### PR TITLE
OpenAI features IGs.

### PR DESCRIPTION
**Pull request by ilyakubanov**
Ticket: https://spryker.atlassian.net/browse/CC-34800

### CODE REVIEW

File: _includes/pbc/all/install-features/202410.0/install-the-image-search-ai-module.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+search.with.your-images.no-results,"Keine Ergebnisse gefunden. Bitte versuchen Sie es mit einem anderen Bild.",de_DE
+```
+
+### 3) Enable controllers
+
+Register the following route providers on the Storefront:
+
+| PROVIDER                              | NAMESPACE                                   |
+|---------------------------------------|---------------------------------------------|
+| ServicePointWidgetRouteProviderPlugin | SprykerEco\Yves\ImageSearchAi\Plugin\Router |
```
Comment (dmiseev) [inline _includes/pbc/all/install-features/202410.0/install-the-image-search-ai-module.md]: ServicePoint?

File: _includes/pbc/all/install-features/202410.0/install-the-product-management-ai-module.md [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+
+Make sure the following modules have been installed:
+
+| MODULE              | EXPECTED DIRECTORY                       |
+|---------------------|------------------------------------------|
+| ProductManagementAi | vendor/spryker-eco/product-management-ai |
+
+{% endinfo_block %}
+
+### 2) Set up database schema and transfer objects
```
Comment (dmiseev) [inline _includes/pbc/all/install-features/202410.0/install-the-product-management-ai-module.md:36]: For projects that don't have  `ProductMerchantPortalGui` module we still need to extend ProductAbstractTransfer:

```
<transfer name="ProductAbstract">
    <property name="categoryIds" type="int[]" singular="idCategory"/>
</transfer>

File: _includes/pbc/all/install-features/202410.0/install-the-product-management-ai-module.md [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+| spy_product_image.alt_text | column | created |
+
+Make sure the following changes have been applied in transfer objects:
+
+| TRANSFER                     | TYPE     | EVENT   | PATH                                                       |
+|------------------------------|----------|---------|------------------------------------------------------------|
+| AiTranslatorRequest          | class    | created | src/Generated/Shared/Transfer/AiTranslatorRequestTransfer  |
+| AiTranslatorResponseTransfer | class    | created | src/Generated/Shared/Transfer/AiTranslatorResponseTransfer |
+| ProductImage.altText         | property | created | src/Generated/Shared/Transfer/ProductImageTransfer         |
+| ProductImageStorage.altText  | property | created | src/Generated/Shared/Transfer/ProductImageStorageTransfer  |
```
Comment (dmiseev) [inline _includes/pbc/all/install-features/202410.0/install-the-product-management-ai-module.md:60]: Also add here `ProductAbstract.categoryIds`

File: _includes/pbc/all/install-features/202410.0/install-the-image-search-ai-module.md (hunk 1 above)
Comment (ilyakubanov) [reply _includes/pbc/all/install-features/202410.0/install-the-image-search-ai-module.md]: Fixed

File: _includes/pbc/all/install-features/202410.0/install-the-product-management-ai-module.md (hunk 2 above)
Comment (ilyakubanov) [reply _includes/pbc/all/install-features/202410.0/install-the-product-management-ai-module.md:36]: It is in module definition already: [install-the-product-management-ai-module.md](https://github.com/spryker/spryker-docs/pull/2905/files/8b06c3b7649a69f7896806b906a9847b6233d61b#diff-91b842efdcd2728293c61b16d105ffcba80342a0d052a8af42187c73938e4272)

File: _includes/pbc/all/install-features/202410.0/install-the-product-management-ai-module.md [hunk 4]
```diff
@@ -58,6 +58,7 @@ Make sure the following changes have been applied in transfer objects:
 | AiTranslatorResponseTransfer | class    | created | src/Generated/Shared/Transfer/AiTranslatorResponseTransfer |
 | ProductImage.altText         | property | created | src/Generated/Shared/Transfer/ProductImageTransfer         |
 | ProductImageStorage.altText  | property | created | src/Generated/Shared/Transfer/ProductImageStorageTransfer  |
+| ProductAbstract.categoryIds  | property | created | src/Generated/Shared/Transfer/ProductAbstract Transfer     |
```
Comment (dmiseev) [inline _includes/pbc/all/install-features/202410.0/install-the-product-management-ai-module.md:61]: src/Generated/Shared/Transfer/ProductAbstractTransfer

**Review by dmiseev — DISMISSED** [review]

File: docs/pbc/all/search/202410.0/base-shop/third-party-integrations/visual-search-powered-by-openai/install-visual-search-powered-by-openai.md [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+    protected function getGlobalWidgets(): array
+    {
+        return [
+            ImageSearchAiWidget::class,
+        ];
+    }
+}
+```
+
+See storefront related code in the demoshops PRs as an example of how to integrate the ImageSearchAi module frontend into your project:
```
Comment (andriitserkovnyi) [inline docs/pbc/all/search/202410.0/base-shop/third-party-integrations/visual-search-powered-by-openai/install-visual-search-powered-by-openai.md:131]: are there additional steps to do or is it the same as described in the document?

**Pull request merged**